cmd/composectl/cmd/compose.go

```diff
@@ -0,0 +1,61 @@
+package composectl
+
+import (
+	"fmt"
+	"github.com/containerd/containerd/platforms"
+	"github.com/foundriesio/composeapp/pkg/compose"
+	v1 "github.com/foundriesio/composeapp/pkg/compose/v1"
+	"github.com/spf13/cobra"
+	"path"
+)
+
+var (
+	composeCmd = &cobra.Command{
+		Use:   "compose",
```

Comment:
Any reason to omit <ref> from the usage string?
```suggestion
		Use:   "compose <ref>",
```